### ISSUE TITLE
How to use ChatGLM for Masked language modeling?

### ISSUE THREAD

**Issue by zlq147** [THUDM/ChatGLM-6B#242]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

How to use ChatGLM for Masked language modeling? Now I would like to get the distribution of [mask] position, e.g. "Three hours of test, I am [mask].". How can I use ChatGLM in huggingface?

### Expected Behavior

Expect to get the logit of words.

### Steps To Reproduce

Not yet.

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

### Anything else?

Not yet.

**Comment by duzx16** [comment]
Not supported.
